core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -350,6 +350,11 @@ object Schema extends LowPrioritySchema with SchemaCompanionMacros {
     val Attribute: AttributeKey[UniqueItems] = new AttributeKey[UniqueItems]("sttp.tapir.Schema.UniqueItems")
   }
 
+  case class ProductAsArray(productAsArray: Boolean)
```

Comment:
maybe we could add a comment here, explaining what is the purpose of this attribute, or at least linking to this issue number (#3941)?